app/retail/emails.py

```diff
@@ -932,8 +932,8 @@ def render_start_work_applicant_expired(interest, bounty):
 def render_new_bounty_roundup(to_email):
     from dashboard.models import Bounty
     from django.conf import settings
-    subject = "Grow Ethereum Hackathon: Just Days Away"
-    new_kudos_pks = [153, 66, 4281]
+    subject = "Hacking, Begin! Grow Ethereum Has Started"
```

Comment:
just `The Grow Ethereum  Hackathon Has Started` seems simpler..

Reply:
fixed